### ISSUE TITLE
Smart moving reloaded compatibility

### ISSUE THREAD

**Issue by Zahkriiven** [iwoplaza/MoBends#35]
Hello, as I asked in [here](https://github.com/Tommsy64/SmartMovingReloaded/issues/7), the old mod Smart moving is getting a port and rewrite by Tommsy64 for recent versions of minecraft and as I know this mod is getting a rewrite too I think it is the perfect moment to make them compatible if possible.
I think both mods work pretty well together and adding [Shoulder Surfing Reloaded](https://github.com/Exopandora/ShoulderSurfing) makes the perfect third person gameplay experience.

Thank you.
~Zahkriiven

**Comment by QuerrixTU** [comment]
I agree with you buth the both mod use a differente rendering method and it's would be very difficult to make them compatible
